### PR TITLE
FIX: The Arbiter fault on behalf of qrix is now auto reset.

### PR DESCRIPTION
**Pull request by ghalym**
The Arbiter fault on behalf of qrix is now auto reset. The user have to reset only the actual faults on qrix, when those are cleared this summary fault clears as well.

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: Arbiter/KFEArbiter/LineIDs.dbg
Comment (ZLLentz) [inline Arbiter/KFEArbiter/LineIDs.dbg:1]: .dbg files shouldn't be committed, this repo's gitignore is old: https://confluence.slac.stanford.edu/display/PCDS/TwinCAT+3+Git+Setup+and+Best+Practices#TwinCAT3GitSetupandBestPractices-.gitignore

File: Arbiter/_Config/IO/Device 1 (EtherCAT).xti [hunk 1]
```diff
@@ -51,6 +51,7 @@
 			<SyncUnit Name="tmo" NoDeleteIfUnused="true"/>
 			<SyncUnit Name="crix" NoDeleteIfUnused="true"/>
 			<SyncUnit Name="qrix" NoDeleteIfUnused="true"/>
+			<SyncUnit Name="future" NoDeleteIfUnused="true"/>
```
Comment (ZLLentz) [inline Arbiter/_Config/IO/Device 1 (EtherCAT).xti:54]: This isn't causing a problem, but what is this? Can we document what it is in this PR?

**Review by ZLLentz — DISMISSED** [review]
Code is fine BTW, documentation and gitignore are important too but if you need a merge for PAMM we can follow-up later

**Comment by ghalym** [comment]
> Code is fine BTW, documentation and gitignore are important too but if you need a merge for PAMM we can follow-up later

Thank you. I fixed the gitignore file and ran pre-commit.

File: Arbiter/_Config/IO/Device 1 (EtherCAT).xti (hunk 1 above)
Comment (ZLLentz) [reply Arbiter/_Config/IO/Device 1 (EtherCAT).xti:54]: @ghalym we should document what this is for in the PR description

**Pull request merged**